src/cpp/src/llm_pipeline.cpp

```diff
@@ -385,23 +385,25 @@ class ContinuousBatchingAdapter final : public LLMPipelineImplBase {
         const std::filesystem::path& model_path,
         const Tokenizer& tokenizer,
         const std::string& device,
-        const ov::AnyMap& plugin_config
+        const ov::AnyMap& plugin_config,
+        const SchedulerConfig& scheduler_config
```

Comment:
`scheduler_config` is ignored with the new approach. Remove it

Reply:
A possibility to provide `SchedulerConfig` as `plugin_config` should be described in docstrings. I forgot to do that when I added CB device.